### ISSUE TITLE
Not showing after launching

### ISSUE THREAD

**Issue by FireAntJoe** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1555]
**OS:** Fedora 24 64bit           

**Issue Descriptions:** I click the icon in fedora and it launches but nothing appears on my screen. I check system monitor after and I don't see it running.

**Steps to Reproduce:** Download the latest version of the GPMDP using the latest Fedora 64bit click it.

**Comment by MarshallOfSound** [comment]
>  it launches but nothing appears on my screen. I check system monitor after and I don't see it running.

How do you know it launches if you can't see it running 😕 

> Download the latest version of the GPMDP using the latest Fedora 64bit click it.

I'm not going to add yet another linux VM to my hard drive so the best I can do is ping @jostrander and ask you to try running it from Terminal with the following env vars set

```bash
export ELECTRON_ENABLE_LOGGING=true
export ELECTRON_ENABLE_STACK_DUMPING=true
```

**Comment by FireAntJoe** [comment]
I saw it in the upper bar and I just switched to antergos.

**Comment by jostrander** [comment]
With that being said, is this something to do with your window manager, OS, platform, etc, or a bug in the player?

**Comment by FireAntJoe** [comment]
I think a bug in the player

**Comment by MarshallOfSound** [comment]
> I think a bug in the player

Is there anything to back that up?

Personally I'm 99% sure this is an issue with your Window Manager.  I base this assumption of multiple things.

1. This issue can not be replicated on Windows, macOS or the any of the 5 linux VM's I have
2. You appear to be running a non-standard WM / OS combination
3. No one else has reported this issue

Can you please provide the terminal output I requested before so I can determine for certain if this is your WM or us?

**Comment by FireAntJoe** [comment]
I think you are right it is a wm manager . I will check it out
 independently and get back to you.

**Comment by AdmiralAsshat** [comment]
I just installed the latest Fedora 24 64-bit version (good to mention exactly **which** version for someone reviewing the issue six months from now--I installed 3.5.0), and mine launches without problem.

If you'd like to look more into it, we may need more details about your environment. What desktop/window manager were you running? Mine is currently running Cinnamon, which seems to be having problems.

You mentioned switching to Antergos, which is a flavor of Arch, if I'm not mistaken, but I believe it also uses Cinnamon. Does the Arch build work on your Antergos environment?
